edweiss-app/app/(app)/deck/[id]/index.tsx

```diff
@@ -189,25 +185,27 @@ const CardListScreen: ApplicationRoute = () => {
 
 export default CardListScreen;
 
+/**
+ * The DisplayCard component displays a card with the question and learning status
+ * 
+ * @param Card: Memento.Card 
+ * @param isSelected: boolean indicates if the card is selected
+ * @param toggleSelection: function to toggle card selection
+ * @param onLongPress: function to handle long press
+ * @param selectionMode: boolean indicates if the selection mode is active
+ * @param goToPath: function to navigate to the handle card press
+ * @returns display card component
+ */
 const DisplayCard: ReactComponent<{ card: Memento.Card, isSelected: boolean, toggleSelection: (card: Memento.Card) => void; onLongPress: () => void; selectionMode: boolean; goToPath: () => void; }> = ({ card, isSelected, toggleSelection, onLongPress, selectionMode, goToPath }) => {
```

Comment:
It would be better to put that component in the directory `components/deck/`. I think it is better to not put an application route and a related react component in a same file.

Reply:
It is a good idea to separate totally `ApplicationRoute` and `Component`. However I think is not super necessary because:
1) `DisplayCard` is only for that specific screen and it only to display a simple design box that displays some information about the card
2) I don't export it meaning that it will not interfere with the actual importance thing: `CardListScreen`
3) If not for testing reason, I would prefer grouping things in 1 file. But yes, if it is a bad practice then I will definitely change it.

So given that I will definitely keep in mind your comment because I do agree that it ensures that the only thing that gets exported is `CardListScreen`. However, for the sake of simplicity, I will still keep DisplayCard component in this file.